eq-author/src/components/QuestionPicker/index.test.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+
+import { render, fireEvent } from "tests/utils/rtl";
+
+import QuestionPicker from ".";
+import mockSections from "../../tests/mocks/mockSections.json";
+
+describe("QuestionPicker", () => {
+  const defaultProps = {
+    title: "Select the question(s) to import",
+    isOpen: true,
+    showSearch: true,
+    warningPanel:
+      "You cannot import folders but you can import any questions they contain.",
+    onSubmit: jest.fn(),
+    onClose: jest.fn(),
+    startingSelectedQuestions: [],
+    sections: mockSections,
+  };
+
+  const renderQuestionPicker = (args) =>
+    render(<QuestionPicker isOpen {...defaultProps} {...args} />);
+
+    describe("Testing the render", () => {
+      it("Can render", () => {
+        const { getByText } = renderQuestionPicker({});
+
+        expect(getByText(defaultProps.title)).toBeInTheDocument();
+      });
+
+    });
+
+    describe("Functions Firing", () => {
+      it("onSubmit Fires", () => {
+        const { getByText } = renderQuestionPicker({});
+
+        const selectButton = getByText("Select");
+
+        fireEvent.click(selectButton);
+
+        expect(defaultProps.onSubmit).toHaveBeenCalled();
+      });
+
+      it("onClose Fires", () => {
+        const { getByLabelText } = renderQuestionPicker({});
+
+        const closeButton = getByLabelText('Close');
+
+        fireEvent.click(closeButton);
+
+        // getByLabelText can also grab the aria label
+        expect(defaultProps.onClose).toHaveBeenCalled();
+      });
+    });
+
+    describe("Interaction testing", () => {
+      it("tests clicks on individual questions and registers the clicks", () => {
+        const { getByText } = renderQuestionPicker({});
+
+        // This is a bit strange of a test. I'm grabbing the aria attribute from the closest div
+        // and making an expectation on tha, since the aria is on the div, not the <p>. 
+        // Allows us to check the clicking interactions are
+        // firing, but there may be a better way of doing this.
```

Comment:
```suggestion
```

I think this is probably fine - there may be a nicer way to select for the parent than just the `div` selector (although would need to change the component slightly for that), but no matter what you're going to have to query the parent to find out if the styling / aria attribute has changed.